lib/rubocop/ast/node/args_node.rb

```diff
@@ -24,6 +24,45 @@ class ArgsNode < Node
       def empty_and_without_delimiters?
         loc.expression.nil?
       end
+
+      # Yield each argument from the collection.
+      # Arguments can be inside `mlhs` nodes in the case of destructuring, so this
+      # flattens the collection to just `arg`, `optarg`, `restarg`, `kwarg`,
+      # `kwoptarg`, `kwrestarg` and `blockarg`.
+      #
+      # @overload each_arg
+      #   Yield all arguments.
+      # @overload each_arg(type)
+      #   Yield only arguments matching the type.
+      #   @param [Symbol] type a node type
+      # @overload each_arg(type_a, type_b, ...)
+      #   Yield only arguments matching any of the types.
+      # @yieldparam [Node] node each arg
+      # @return [self] if a block is given
+      # @return [Enumerator] if no block is given
+      def each_arg(*types, &block)
+        if types.empty?
+          types = ARGUMENT_TYPES.dup
+          types << :forward_arg if RuboCop::AST::Builder.emit_forward_arg
+        end
```

Comment:
@marcandre I'm not sure if `ARGUMENT_TYPES` should include `forward_arg`?

Reply:
Yes, please include it